actions/entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -euxo pipefail
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://docs.github.com/en/actions/learn-github-actions/environment-variables
+# GITHUB_EVENT_PATH contains the json file for the event.
+# GITHUB_SHA contains the commit hash.
+# GITHUB_WORKSPACE contains the repo folder.
+# GITHUB_EVENT_NAME contains the event name.
+
+export GITHUB_AUTH_TOKEN="$INPUT_REPO_TOKEN"
+export SCORECARD_V3=1
+export SCORECARD_POLICY_FILE="$INPUT_POLICY_FILE"
+export SCORECARD_SARIF_FILE="$INPUT_SARIF_FILE"
```

Comment:
Could you explain why users need to provide a SARIF file location? Isn't this something internal that will be used by GitHub and need not be exposed to users?

Reply:
see https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/uploading-a-sarif-file-to-github
We can make it internal or let users decide. Note that SARIF option is public (`--format sarif`).
 
ShiftLeftSecurity does the same, see https://github.com/ShiftLeftSecurity/scan-action - I think they do it this way because it allows them to print results in a different format for users, which we could possibly too.

@naveensrinivasan any opinion on this?

Reply:
Right. But there isn't a need to ask the user to supply us with the SARIF file path. That can be internal. ShiftLeft seems to be doing the same thing.

Reply:
sorry, shiftleft was the wrong example, I mis-read upload-artifact for upload-sarif. It's up to us really. GH lets users do it with their built-in action in the workflow, as explained https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/uploading-a-sarif-file-to-github
Nothing stopping us from doing inside our action. I just picked the simplest solution for the first iteration.